### DISCUSSION TITLE
Override excerpt metadata

### DISCUSSION THREAD

**Discussion by Phil89a** [Pagefind/pagefind#673] [Q&A]
Is it possible to override the excerpt that PageFind uses to display under the title in the Search UI results?

thanks

**Comment by bglw** [comment]
Hi @Phil89a 👋 

Sorry for the delay, I missed this one come through discussions.
In case you're still looking, you could play around with the [processResult hook of the Default UI](https://pagefind.app/docs/ui/#process-result), or alternatively look into the [WIP Modular UI](https://www.npmjs.com/package/@pagefind/modular-ui) which has better template customization for showing results.
